src/main/scala/io/iohk/ethereum/ledger/Ledger.scala

```diff
@@ -136,6 +138,30 @@ class LedgerImpl(vm: VM, blockchainConfig: BlockchainConfig) extends Ledger with
         }
   }
 
+  override def simulateTransaction(stx: SignedTransaction, blockHeader: BlockHeader, storages: BlockchainStorages, validators: Validators): TxResult = {
```

Comment:
The `validators` parameter seems to currently not being used, should we start using it or maybe remove it?